interface/src/scripting/AssetMappingsScriptingInterface.h

```diff
@@ -64,6 +64,7 @@ class AssetMappingsScriptingInterface : public QObject, public Dependency {
 
     Q_INVOKABLE void setMapping(QString path, QString hash, QJSValue callback);
     Q_INVOKABLE void getMapping(QString path, QJSValue callback);
```

Comment:
you may want to add defaults for these, I don't think you'll be able to call from the JS side without them unless they have a default value